test/src/router-resolution.js

```diff
@@ -60,27 +64,43 @@ test('router-resolution', async (t) => {
 
   for (const testCase of cases) {
     const requestUrl = dirname + testCase.url
-    const response = await fetch(requestUrl)
+    const response = await fetch(requestUrl, { redirect: 'manual' })
```

Comment:
what is `{ redirect: 'manual' }`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/fetch#redirect